### ISSUE TITLE
Quick question on atlas database url

### ISSUE THREAD

**Issue by d3287t328** [AntonioErdeljac/twitter-clone#2]
For the value DATABASE_URL is the correct url look like this?

```
mongodb+srv://user:password@cluster0.string.mongodb.net/?retryWrites=true&w=majority
```

**Comment by JKomieter** [comment]
> For the value DATABASE_URL is the correct url look like this?
> 
> ```
> mongodb+srv://user:password@cluster0.string.mongodb.net/?retryWrites=true&w=majority
> ```

Yes, changing the the user to your username and the password to the password you used for database.

**Comment by MostlyKIGuess** [comment]
afaik it's only upto .net/
do you really need retry writes true? I am not super sure
